deploy/charts/emqx/values.yaml

```diff
@@ -146,7 +146,7 @@ ingress:
 
   ## ingress for EMQX Mgmt API
   mgmt:
-    enabled: false
+    enabled: true
```

Comment:
I think we should leave it at the default value of false, because not everyone will use the ingress